package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Oran.js JavaScript by Orange Cats",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint",
+    "pretest": "npm run lint",
+    "test": "mocha",
```

Comment:
You can leave `eslint .` there and add mocha to the script like this `eslint . && mocha` so you don't have to put the pretest script in there.  Then we don't need to add anything to the pre-commit array.